plugins/foreman_ansible/2.x/index.md

```diff
@@ -19,42 +19,124 @@ You can define a list of roles to enforce, or just run arbitrary playbooks on th
-
-To obtain a valid certificate for your host follow the procedure as described [here](http://theforeman.org/manuals/latest/index.html#4.3.10SSL)
-See the [python-requests documentation](http://docs.python-requests.org/en/master/user/advanced/#ssl-cert-verification) on the details of certificate setup.
-
-* If you use Ansible 2.2+ , our callback is installed with Ansible itself, simply change `/etc/ansible/ansible.cfg` to contain
-
-      [defaults]
-      bin_ansible_callbacks = True
-      callback_whitelist = foreman
-
+<script type="text/javascript">
+function update_ansible_version(select) {
+  var version = select.value;
+  $(".ansible_version").hide();
+  if (version && version != 'none') {
+    $(".ansible_version_"+version).show();
+  } else {
+    $(".ansible_version_none").show();
+  }
+}
+</script>
+
+In order to make Ansible send us data from the hosts, we set up a callback on your host that runs Ansible.
+To provide specific nstallation instructions, please select your Ansible version:
+
+<select onChange="update_ansible_version(this);">
+  <option value="none">-- select Ansible version --</option>
+  <option value="25">2.5 or higher</option>
+  <option value="22">2.2, 2.3, 2.4</option>
+  <option value="21">2.1 or lower</option>
+</select>
+
+<div class="ansible_version ansible_version_none">
+  <i>No Ansible version selected.</i>
+</div>
+
+
+<div class="ansible_version ansible_version_25 ansible_version_22" style="display:none;">
+<p>
+Our callback is installed with Ansible itself, simply change <code>/etc/ansible/ansible.cfg</code> to contain
+</p>
+
+{% highlight bash %}
+[defaults]
+callback_whitelist = foreman
+{% endhighlight %}
+
+<p>
 and the callback will be enabled after that.
-
-
-For Ansible versions older than 2.2:
-
-* Change your `/etc/ansible/ansible.cfg` to contain the following options
-
-      callback_plugins = ~/.ansible/plugins/callback_plugins/
-      bin_ansible_callbacks = True
-
-* And run this line to download the callback file in the appropriate location
-
-      wget https://github.com/ansible/ansible/blob/devel/lib/ansible/plugins/callback/foreman.py -O ~/.ansible/plugins/callback_plugins/foreman.py
-
-* Alternatively, you can find the source for the [Ansible callback here](https://github.com/ansible/ansible/blob/devel/lib/ansible/plugins/callback/foreman.py) and deploy it in your directory of choice.
+</p>
+
+<p>
+Of course, the callback cannot know automatically where is Foreman located and which certificates to use in its requests.
+</p>
+</div>
+
+<div class="ansible_version ansible_version_25" style="display:none;">
+<p>
+To configure it, simply add a section <code>[callback_foreman]</code> at the end of <code>/etc/ansible/ansible.cfg</code> like this:
+</p>
+
+{% highlight bash %}
+[callback_foreman]
+url = 'https://centos7-devel.lobatolan.home'
+ssl_cert = /home/dlobatog/workspace/ansible-demo/katello-default-ca.crt
```

Comment:
Perhaps point this to the default foreman/katello certs? You should also be able to point it to the CA and have it verified.